webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -287,18 +261,17 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     setIsLoading(true);
     setPage(0);
     setExperiments(INITIAL_LOADING_EXPERIMENTS);
+    setSelection({ columns: CompactSelection.empty(), rows: CompactSelection.empty() });
```

Comment:
shouldn't this also reset the id selection?

Reply:
I left this logic alone. The reason the ids are NOT reset here is because it supports preserving selection for when user change the column sort order and table mode (infinite vs paginated).